tools/BuildPackagesTask/Microsoft.Azure.Build.Tasks/CIFilterTask.cs

```diff
@@ -193,8 +193,7 @@ private List<string> GetDependentModuleList(string moduleName, Dictionary<string
         // Run a selected module list instead of run all the modules to speed up the CI process.
         private List<string> GetSelectedModuleList()
         {
-            string[] lines = System.IO.File.ReadAllLines(@"./tools/BuildPackagesTask/Microsoft.Azure.Build.Tasks/SelectedModuleList.txt");
-            return new List<string>(lines);
+            return new List<string>(Environment.GetEnvironmentVariable("SELECTEDMODULELIST").Split(';'));
```

Comment:
@wyunchi-ms Hi Yunchi, I got a null reference error on this line when trying to build locally by `dotnet build .\build.proj /p:TargetModule=Accounts`. Can you take a look?